### PR TITLE
fix nulti-tooltip and date picker component

### PR DESCRIPTION
**Pull request by Pycomet-zz**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -29,28 +31,30 @@ class App extends Component {
...
-              </div>
-              <div className="m-6">
-                <div className="guide-1">
-                  <GroupedButton
-                    showGlobe={this.state.showGlobe}
-                    toggleDisplay={this.toggleDisplay}
-                  />
+      <MuiPickersUtilsProvider utils={DateFnsUtils}>
+        <MuiThemeProvider theme={theme}>
+          <NavigtaionContext.Provider
```
Comment (chichengren) [inline src/App.js]: `NavigationContext` is already provided in the top level, I don't think we need to define it here again

**Review by chichengren — CHANGES_REQUESTED** [review]
@Pycomet Can you also run `yarn lint:fix` to fix the jsx styling incosistency?

File: src/App.js (hunk 1 above)
Comment (Pycomet-zz) [reply src/App.js]: Resolved!

**Comment by sidgan** [comment]
@chichengren can you confirm if this PR is still valid? Else lets close it.

**Comment by chichengren** [comment]
Closing this PR in favor of previous merged one

**Pull request closed without merging**